src/matrix/ssss/common.ts

```diff
@@ -14,25 +14,42 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+import type {Platform} from "../../platform/web/Platform.js";
+
+export interface IKeyDescription {
```

Comment:
I'd use `type` here, see https://github.com/vector-im/hydrogen-web/blob/master/doc/TS-MIGRATION.md#use-type-rather-than-interface-for-named-parameters-and-pojo-return-values
```suggestion
export type KeyDescriptionData = {
```